src/factories/serviceFactory.js

```diff
@@ -6,6 +6,10 @@ const repositories = {
   contract: {
     serviceLoader: () => import("@/services/contract.js"),
     formComponent: () => import("@/components/contracts/ContractForm.vue")
+  },
+  renewContract: {
```

Comment:
renew vs. extend - gibt es einen Unterschied? Ansonsten können wir es auch konsistent benennen (extend finde ich dann besser)

Reply:
Ansonsten bin ich an der Stelle nicht ganz sicher, was die Factory tut - sieht aber aus, als bräuchten wir das.
(Es gibt aber einige Residuen, die nie aufgeräumt wurden und möglicherweise obsolet sind... Ich habe in diesen factories noch nie etwas geändert)

Reply:
Die Factory baut die Form Dialoge.

Reply:
Laut dict.cc https://www.google.de/search?hl=de&q=verl%E4ngern%20-dict.cc ist "renew" die passende Überstzung für "verlängern" im Vertragskontext

Reply:
Then let's make it so